README.md

```diff
@@ -1,16 +1,41 @@
 # MOJ Gutenberg Blocks
+MOJ Gutenberg Blocks is a companion WP block plugin to be installed with the WP and the Hale theme. It allows for greater customisation and enhancements unique to our websites.
 
-This repository houses the MOJ Gutenberg blocks.
- 
-## Development Instructions
-To develop your own modifications, you will need to download the full repo - ideally you should be in your `wp-content/plugins` folder.
+## Features
+This plugin adds the following blocks,
+* Accordion
+* Card
+* CTA
+* Hero
+* Highlights-list
+* Intro
+* Quote
+* Reveal
+* Staggered-boxes
+* Video
```

Comment:
Maybe worth a note saying Video is a placeholder..theres no block

Reply:
Ah, okay I didn't realise video was a placeholder, I might remove completely then?

Reply:
Removed video block for now